prereqs_mac.sh

```diff
@@ -3,17 +3,16 @@
 which -s brew
 if [[ $? != 0 ]] ; then
     # Install Homebrew
-    echo "Donwloading and installing homebrew"
+    echo "Downloading and installing homebrew"
```

Comment:
Nice touch! Really going the extra mile here!

Reply:
haha